server/odc-service/src/main/java/com/oceanbase/odc/service/connection/database/DatabaseService.java

```diff
@@ -423,6 +423,7 @@ public Boolean internalSyncDataSourceSchemas(@NonNull Long dataSourceId) throws
                             database.setCollationName(latest.getCollationName());
                             database.setCharsetName(latest.getCharsetName());
                             database.setLastSyncTime(latest.getLastSyncTime());
+                            database.setExisted(Boolean.TRUE);
```

Comment:
Does this fix have a related issue?

Reply:
yes, For mysql and ODP mysql does not have a unique identifier for the database, fix that when a database is deleted and then recreated, it will always show the status of being deleted and will not be updated。